src/Base.sol

```diff
@@ -21,6 +22,7 @@ abstract contract CommonBase {
 
     Vm internal constant vm = Vm(VM_ADDRESS);
     StdStorage internal stdstore;
+    Console internal console;
```

Comment:
Hmm so this will technically be a breaking change for anyone who was importing the original `conosle.sol`, since there a collision with that library and this variable. But in solidity, you should never need the original `console.sol` (it's just for hardhat users) and should always prefer `console2`, so I think this is ok. 

It also means you can now always just use `console.log` instead of `console2.log`, but get all the benefits of `console2` which is nice.

Wdyt @gakonst @ZeroEkkusu?

Reply:
Yeah, I thought of this when developing, but thought this would be the best approach in terms of user experience. Alternatively, this could be set to another word, eg: print. But I think this will be more confusing for users who are already used to do the console.log